indexing-service/src/test/java/org/apache/druid/indexing/common/task/IndexTaskSerdeTest.java

```diff
@@ -267,6 +272,7 @@ public void testBestEffortRollupWithHashedPartitionsSpec()
         true,
         10,
         100,
+        null,
         null
```

Comment:
Do we need an `IndexTaskSerdeTest` where `awaitSegmentAvailabilityTimeoutMillis` is not null

Reply:
there was not any new explicit test written the other tests in that file do use -1L and 1L as test values. I just updated another of the existing tests to use 0L as well.